src/Mailgun/Resource/Api/Domain/DeliverySettingsUpdateResponse.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/**
+ * Copyright (C) 2013-2016 Mailgun.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+namespace Mailgun\Resource\Api\Domain;
+
+use Mailgun\Assert;
+use Mailgun\Resource\CreatableFromArray;
+use Mailgun\Resource\Api\SimpleResponse;
+
+/**
+ * @author Sean Johnson <sean@mailgun.com>
+ */
+class DeliverySettingsUpdateResponse extends SimpleResponse implements CreatableFromArray
+{
+    /**
+     * @var bool
+     */
+    private $noVerify;
+
+    /**
+     * @var bool
+     */
+    private $requireTLS;
+
+    /**
+     * @param array $data
+     *
+     * @return SettingsUpdateResponse|array|ResponseInterface
+     */
+    public static function createFromArray(array $data)
+    {
+        $message = array_key_exists('message', $data) ? $data['message'] : null;
+        $noVerify = array_key_exists('skip_verification', $data) ? $data['skip_verification'] : null;
+        $requireTLS = array_key_exists('require_tls', $data) ? $data['require_tls'] : null;
```

Comment:
❓ Why are not setting here a `bool` as by default value, instead of `null`? IMO should be nice if the SDK provides the best/secure option by default and `null` is not giving value from my POV.